BusyList/ITaskRepository.cs

```diff
@@ -2,9 +2,16 @@
 
 namespace BusyList
 {
+    public enum TaskStatus
+    {
+        Done,
+        InProcess,
+        NotStarted
```

Comment:
This should be the first item, as the default enum value is in the first position, and it makes the most sense for NotStarted to be the default.

Reply:
Ohh yeah ofc